pkg/command/install/verison.go

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 Singularity Data
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package install
+
+import "fmt"
+
+func minimumVersion(versionStr string) string {
+	version := formatVersion(versionStr)
+	if version < MinimumVersion {
```

Comment:
It's not robust. We need to compare the semantic version instead.

I think there're many libs that can help, e.g.,
https://pkg.go.dev/golang.org/x/mod/semver